app/gateway/changelog.md

```diff
@@ -9,6 +9,431 @@ Changelog for supported Kong Gateway versions.
+  * Fixed some timer-related issues where the counter syncing timer couldn't be created or destroyed properly.
+  * The plugin now creates counter syncing timers during plugin execution instead of plugin creation to reduce some meaningless error logs.
+  * Fixed an issue where {{site.base_gateway}} produced a log of error log entries when multiple Rate Limiting Advanced plugins shared the same namespace.
+
+* [**Response Rate Limiting**](/hub/kong-inc/response-ratelimiting/) (`response-ratelimiting`)
+  * Fixed migration of Redis configuration.
+
+* [**DeGraphQL**](/hub/kong-inc/degraphql/) (`degraphql`)
+  * Fixed an issue where GraphQL variables were not being correctly parsed and coerced into their defined types.
+  * This plugin now uses a new configuration handler to update the GraphQL router with better error handling.
+
+* [**LDAP Authentication Advanced**](/hub/kong-inc/ldap-auth-advanced/) (`ldap-auth-advanced`)
+  * Fixed an issue where, if the credential was encoded with no username, Kong Gateway threw an error and returned a 500 code.
+  * Fixed an issue where an exception would be thrown when LDAP search failed.
+
+* [**OAS Validation**](/hub/kong-inc/oas-validation/) (`oas-validation`), 
+[**WebSocket Size Limit**](/hub/kong-inc/websocket-size-limit/) (`websocket-size-limit`), 
+[**WebSocket Validator**](/hub/kong-inc/websocket-validator/) (`websocket-validator`),
+ [**XML Threat Protection**](/hub/kong-inc/xml-threat-protection/) (`xml-threat-protection`)
+  * The [priorities](/gateway/latest/plugin-development/custom-logic/#plugins-execution-order) of these plugins have been updated to prevent collisions between plugins.
+    The relative priority (and the order of execution) of bundled plugins remains unchanged.
+
+
+### Dependencies
+
+* Bumped `atc-router` from v1.6.0 to v1.6.2
+ [#12231](https://github.com/Kong/kong/issues/12231)
+* Bumped `libexpat` to 2.6.2
+ [#12910](https://github.com/Kong/kong/issues/12910)
+* Bumped `lua-kong-nginx-module` from 0.8.0 to 0.11.0
+ [#12752](https://github.com/Kong/kong/issues/12752)
+* Bumped `lua-protobuf` to 0.5.1
+ [#12834](https://github.com/Kong/kong/issues/12834)
+* Bumped `lua-resty-acme` to 0.13.0
+ [#12909](https://github.com/Kong/kong/issues/12909)
+* Bumped `lua-resty-aws` from 1.3.6 to 1.4.1
+ [#12846](https://github.com/Kong/kong/issues/12846)
+* Bumped `lua-resty-lmdb` from 1.4.1 to 1.4.2
+ [#12786](https://github.com/Kong/kong/issues/12786)
+* Bumped `lua-resty-openssl` from 1.2.0 to 1.3.1
+ [#12665](https://github.com/Kong/kong/issues/12665)
+* Bumped `lua-resty-timer-ng` to 0.2.7
+ [#12756](https://github.com/Kong/kong/issues/12756)
+* Bumped PCRE from the legacy `libpcre` 8.45 to `libpcre2` 10.43
+ [#12366](https://github.com/Kong/kong/issues/12366)
+* Bumped `penlight` to 1.14.0
+ [#12862](https://github.com/Kong/kong/issues/12862)
+* Added the package `tzdata` to the DEB Docker image for convenient timezone setting
```

Comment:
This one seems like a fix instead of a dependency?

Reply:
It's adding a new dependency named `tzdata`. We don't normally have too many changelog entries like this, so it does look out of place, but there's no real other place for it to go.